src/utils.js

```diff
@@ -22,4 +22,10 @@ const getCategories = () => {
   })
 }
 
-export { getReviews, getCategories, getReviewById };
+const getCommentsByReviewId = (review_id) => {
+  return api.get(`/reviews/${review_id}/comments`).then(({data}) => {
+    return data.comments;
```

Comment:
Great to see you're just returning the data you need in this function so theres no object diving in the comments component.